### ISSUE TITLE
Navball is set to orbit mode on vessel launch from Mesbin.

### ISSUE THREAD

**Issue by WhirligigGirl** [GregroxMun/Whirligig-World#8]
When you launch a vessel, the navball speed indicator is set to orbit mode, adding around 10 km/s to the surface speed. This requires the user to set the navball to surface speed to use it in-flight.

This was originally a bug, but I have decided it's actually a good feature to have.

* It introduces the player right off the bat to the insane rotational velocity of Mesbin. This is invaluable thematically and mechanically.
* Launching rockets to orbit makes best use of orbital speed anyway, surface speed is only useful for short hops and landings.
* If you don't change to orbital speed, you may find yourself going retrograde when you mean to go prograde, above mesbin-stationary orbit. Surface velocity above the orbit of statmun gets *really weird*. 
* In short, it forces the player to know and care about the difference between these modes.
